tests/test-global-dependency.c

```diff
@@ -3,44 +3,44 @@
 // It also tests if the referances to the a and cc functions
 // will be preserved properly when moving.
 
-#include<stdio.h>
+/*
+get-ir:
+    status: success
```

Comment:
It turns out that you don't need to say `status: success` (I'd forgotten this until Edd reminded me in a recent PR!). So this can just be:
```
get-ir:
split:
build:
run:
  stdout: ...
```

Reply:
Oh, but then can I just skip all the stages and write only: 
```
run:
    stdoutL ...
```
As it says that the default is success anyway?

Reply:
Honestly, I can't remember! Can you try and let me know? I should probably update the documentation to be clearer!

Reply:
This lines here indicate that the default values are used even when command names are not specified in the file's comments.
https://github.com/softdevteam/lang_tester/blob/master/src/tester.rs#L684-L686

Reply:
I would suggest you try it (e.g. remove all the stages, then make one fail) and see what happens. *I* struggle to read lang_tester's code now: it's evolved far further than I initially expected!

Reply:
Removed all of them in: e422905

Reply:
It fails and stops at the correct command., i tried it.

Reply:
Excellent!